vet.sh

```diff
@@ -37,7 +37,8 @@ if [[ "$1" = "-install" ]]; then
   go install \
     golang.org/x/tools/cmd/goimports \
     honnef.co/go/tools/cmd/staticcheck \
-    github.com/client9/misspell/cmd/misspell
+    github.com/client9/misspell/cmd/misspell \
+    google.golang.org/grpc/cmd/protoc-gen-go-grpc
```

Comment:
@dfawley 's comment was actually referring to the `protoc` install thats done from source right below. You could just run `./vet.sh -install` - to install the version of protoc used by Github Actions.

Reply:
@arvindbr8 thank you for detailed explanation, I have updated pipeline and few more things, to make this be good